build.sh

```diff
@@ -98,12 +98,12 @@ do_build_conversion() {
     #${CONTAINER_MGMT} push quay.io/nyoxi/kubevirt-conversion:latest
 }
 
-do_build_vmware() {
+do_build_provider() {
     TAG="v$KUBEVIRT_VERSION-$KUBEVIRT_VMWARE_RELEASE"
```

Comment:
can we change this variable as well, or is somebody dependending on it?

Reply:
Will update